charts/mattermost-team-edition/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: Mattermost Team Edition server.
 name: mattermost-team-edition
-version: 3.13.1
-appVersion: 5.25.1
+version: 3.13.2
+appVersion: 5.25.0
```

Comment:
sorry this should stay in 5.25.1

Reply:
Sorry. Resolved now. Have kept chart version unchanged.